subworkflows/local/cazyme.nf

```diff
@@ -0,0 +1,68 @@
+/*
+    Run rundbcan screening tools
+*/
+
+include { RUNDBCAN_DATABASE         } from '../../modules/nf-core/rundbcan/database/main'
+include { RUNDBCAN_CAZYMEANNOTATION } from '../../modules/nf-core/rundbcan/cazymeannotation/main'
+include { RUNDBCAN_EASYCGC          } from '../../modules/nf-core/rundbcan/easycgc/main'
+include { RUNDBCAN_EASYSUBSTRATE    } from '../../modules/nf-core/rundbcan/easysubstrate/main'
+
+
+workflow CAZYME {
+
+    take:
+    faas    // tuple val(meta), path(PROKKA/PRODIGAL.out.faa)
+    gffs    // tuple val(meta), path(ANNOTATION_ANNOTATION_TOOL.out.gff)
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    // When adding new tool that requires FAA, make sure to update conditions
+    // in funcscan.nf around annotation and dbCAN subworkflow execution
+    // to ensure annotation is executed!
+    ch_faas_for_rundbcan = faas
+    ch_gffs_for_rundbcan = gffs
+
+    // Download dbCAN database
+    RUNDBCAN_DATABASE ()
```

Comment:
All databases should be optionally supplied to the pipeline, so this should be wrapped in an if/else statement (i.e., if someone wants to run on an offline servetr)